test_conformance/c11_atomics/common.h

```diff
@@ -1521,15 +1521,14 @@ int CBasicTest<HostAtomicType, HostDataType>::ExecuteSingleTest(
     if (UseSVM())
     {
         // the buffer object must first be released before the SVM buffer is
-        // freed
-        error = clReleaseMemObject(streams[0]);
+        // freed. Assignment value 0 runs Wrapper Class method reset() defined
+        // in typeWrappers.h which release old object
         streams[0] = 0;
         test_error(error, "clReleaseMemObject failed");
```

Comment:
Minor: After this change there is no error code from `clReleaseMemObject` to check, so this can be removed (see similar case below also).